README.md

```diff
@@ -109,6 +140,9 @@ If you are cross-compiling for macOS using [osxcross](https://github.com/tpoecht
 RUSTFLAGS="-l framework=WebKit" cargo build --target=x86_64-apple-darwin --release
 ```
 
+> [!IMPORTANT]
+> If `nix` and/if `nix-darwin` are used then follow instructions from [Nix Section](#nix--nixos).
```

Comment:
Nix section uses webkit2gtk where on macOS we use the included WKWebView, let's remove this